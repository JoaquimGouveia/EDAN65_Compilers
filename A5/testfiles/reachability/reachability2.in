int main() {
    return f();
}

int f() {
    return 2;
}

int g() {
    return 3;
}

int h() {
    return f();
}