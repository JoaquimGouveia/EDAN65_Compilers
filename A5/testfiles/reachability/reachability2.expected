Function: main
f
---------------
Function: f
---------------
Function: g
---------------
Function: h
f
---------------
